packages/cli/src/commands/start.ts

```diff
@@ -133,8 +134,13 @@ export class Start extends BaseCommand {
 				executingWorkflows = activeExecutionsInstance.getActiveExecutions();
 			}
 
-			//finally shut down Event Bus
+			// Shut down Event Bus
 			await eventBus.close();
+
+			// Finally shut down License manager to unclaim any floating entitlements.
+			// Note: In case of above shutdown activities taking too long, the next line might not get executed.
+			// This is fine, as all entitlements will still be claimed on the next startup, which is the desired behavior in most cases.
+			await Container.get(License).shutdown();
```

Comment:
Can we please still move this to line 100, to give this a higher chance of success?

Reply:
I believe it's prudent to perform this action right at the end of the `stopProcess()` function. This approach ensures that enterprise features, which might still be necessary for completing ongoing workflow executions and other tasks, aren't prematurely disabled.

Reply:
could we notify the license server at the start of `stopProcess`, and not update the in-memory feature flags at all?

Reply:
Good idea! That could work. Let me look into that.